openupgrade_scripts/scripts/product_expiry/14.0.1.0/post-migration.py

```diff
@@ -29,12 +29,9 @@ def migrate(env, version):
         """
         UPDATE stock_move_line sml
         SET expiration_date = spl.expiration_date
-        FROM stock_production_lot spl, product_product pp
-        JOIN product_template pt ON pp.product_tmpl_id = pt.id, stock_picking sp
-        JOIN stock_picking_type spt ON sp.picking_type_id = spt.id
-        WHERE spl.id = sml.lot_id AND sml.picking_id = sp.id
-            AND spt.use_existing_lots AND sml.product_id = pp.id
-            AND pt.use_expiration_date""",
+        FROM stock_production_lot spl
+        WHERE spl.id = sml.lot_id AND spl.expiration_date IS NOT NULL
```

Comment:
why too short of conditions here? 🤔 can you please point to the corresponding odoo code?

Reply:
https://github.com/odoo/odoo/blob/14.0/addons/product_expiry/models/stock_move_line.py#L20-L21

The first IF has not many conditions, I think it could be better to keep it simple :smile: